Client/Application/AppDelegate.swift

```diff
@@ -403,7 +379,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // Pause file downloads.
         // TODO: iOS 13 needs to iterate all the BVCs.
-        BrowserViewController.foregroundBVC().downloadQueue.pauseAll()
+        browserViewController.downloadQueue.pauseAll()
```

Comment:
SHould this be like above: `DownloadQueue().pauseAll()`??

Reply:
I couldn't pinpoint exactly where, but I'm guessing there's downloads enqueued on BVC downloadQueue. And I made a change above where I create a new DownloadQueue and resumeAll there instead of resuming all on BVC's.

I might've overlooked something important there - so just reverting that one liner until we find a way to share one `DownloadQueue` (could this be a candidate for our DI container?)